### PR TITLE
merge office-addin-taskpane-sso

### PR DESCRIPTION
**Pull request by millerds**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,7 +1,3 @@
 {
-    "eslint.validate": [
-        "javascript",
-        "javascriptreact",
-        "typescript"
-    ]
+  "eslint.validate": ["javascript", "javascriptreact", "typescript"]
```
Comment (akrantz) [inline .vscode/settings.json:2]: why are there formatting differences here?

File: src/taskpane/taskpane.js [hunk 2]
```diff
@@ -3,8 +3,7 @@
  * See LICENSE in the project root for license information.
  */
 
-/* global require */
-const excel = require("./excel");
-const outlook = require("./outlook");
-const powerpoint = require("./powerpoint");
-const word = require("./word");
+import "./excel";
```
Comment (akrantz) [inline src/taskpane/taskpane.js:6]: For javascript, shouldn't be converting require() to import here.

**Review by akrantz — CHANGES_REQUESTED** [review]
See comments

File: .vscode/settings.json (hunk 1 above)
Comment (millerds) [reply .vscode/settings.json:2]: unifying formatting across project (one of the tools is making this happen) so there is less differences between during merges

File: src/taskpane/taskpane.js (hunk 2 above)
Comment (millerds) [reply src/taskpane/taskpane.js:6]: The linter complains about using require saying the const values are assigned but not used.

Comment (akrantz) [reply src/taskpane/taskpane.js:6]: You can have the require() call without assigning it to the variable.

**Review by akrantz — APPROVED** [review]

**Review by akrantz — APPROVED** [review]

**Pull request merged**